packages/css/.storybook/main.js

```diff
@@ -9,4 +11,29 @@ module.exports = {
     '@storybook/addon-essentials',
     '@storybook/addon-storysource',
   ],
+  webpackFinal: async (config, { configType }) => {
```

Comment:
configType not used

Reply:
I think we no longer need to adapt the webpack configuration with PostCSS 7.  
Normally it should work without

Reply:
No, by default the loader for .css files is using style-loader and css-loader

Cf: https://github.com/storybookjs/storybook/blob/next/lib/core/src/server/manager/manager-webpack.config.ts#L116-L126

It's why I remove the default rule to use a new one, with postcss ;)

Reply:
this is the default config: console.log(config.module.rules[8].use);

```
[
  '/Users/AMEN/workspace/vitamin-web/node_modules/style-loader/dist/cjs.js',
  {
    loader: '/Users/AMEN/workspace/vitamin-web/node_modules/css-loader/dist/cjs.js',
    options: { importLoaders: 1 }
  },
  {
    loader: '/Users/AMEN/workspace/vitamin-web/node_modules/@storybook/core/node_modules/postcss-loader/src/index.js',
    options: { config: [Object] }
  }
]
```
remove node-modules, install and run storybook without this adaptation

Reply:
I tested it and it works fine without :)

Reply:
for https://github.com/storybookjs/storybook/blob/next/lib/core/src/server/manager/manager-webpack.config.ts#L116-L126

I think there is a plugin system with the storybook webpack config

Reply:
Yeah, you are true.

My bad, I forgot to remove the first line, which remove the default css rule. It's working without, and the PR is up-to-date ;)